udemy_enroller/runner.py

```diff
@@ -26,18 +32,30 @@ def _redeem_courses(
 
     while True:
         udemy_course_links = loop.run_until_complete(scrapers.run())
-
+        logger.info(f"Total courses this time: {len(udemy_course_links)}")
         if udemy_course_links:
             for course_link in udemy_course_links:
+                counter_total += 1
```

Comment:
Just on this we might need to improve in future since if we scrape the same course from coursevania and freebiesglobal it will be +2 when I think it should really only be +1
That filtering needs to be done at the scraper manager level though